Server/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const fs = require('fs');
+const fetch = (...args) => import('node-fetch').then(({default: fetch}) => fetch(...args));
+const router= require('./routes/router')
+const dotenv = require('dotenv').config();
+
+const app = express();
+
+app.disable('x-powered-by');
+
+app.set("port", process.env.PORT || 4000);
+
+app.use(express.static(path.join(__dirname, '..', 'client')));
+app.use(router)
+
+
+module.exports = app;
```

Comment:
it is a good style to always put a newline at the end of the file

https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log